src/js/components/MaskedInput/maskedinput.stories.js

```diff
@@ -198,8 +207,111 @@ class IPv4MaskedInput extends Component {
   }
 }
 
+const DropContent = props => {
+  const { date, onSelect, time, onClose, onChange } = props;
+  return (
+    <Box align="center">
+      <Calendar date={date} onSelect={onSelect} />
```

Comment:
```suggestion
      <Calendar date={date} onSelect={onSelect} showAdjacentDays={false} />
```

Reply:
The purpose of this prop suggestion is cleaning up the UI so it will be easier to focus on the MaskedInput.